server/app.js

```diff
@@ -90,11 +89,9 @@ app.post('/verify-token', (req, res) => {
   const { token } = req.body
   jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (err) => {
     if (err) {
-      // res.json({ auth: false })
-      res.send(401, 'Failed to Authenticate')
+      res.status(401).send('Failed to authenticate')
```

Comment:
🙌🏾
Awesome work changing to using status codes. This is the more traditional way of communicating http responses.